src/ape_accounts/accounts.py

```diff
@@ -190,6 +190,8 @@ def sign_message(self, msg: Any, **signer_options) -> Optional[MessageSignature]
             # Message Data
             display_msg += "Message\n"
             for field, value in msg._body_["message"].items():
+                if isinstance(value, bytes):
+                    value = HexBytes(value).hex()
```

Comment:
if we care about 0x prefix, this may no longer be the way to do this once web3.py 7 is out